bin/ghost

```diff
@@ -4,6 +4,6 @@
 // provide a title to the process
 process.title = 'ghost';
 
-var cli = require('../lib/cli');
-
+require('../lib/utils/update-notifier');
+var cli = require('../lib/utils/cli');
```

Comment:
this will break the application as the `cli` file is not in the utils folder - it can be left out of the utils folder since it's listed as the "main" file in package.json